schemas/com.snowplowanalytics.snowplow.media/fullscreen_change_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event fired immediately after the browser switches into or out of full-screen mode.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "fullscreen_change_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "fullscreen": {
+      "type": "boolean",
+      "description": "Whether the video element is fullscreen"
```

Comment:
On this description let's maybe just make it clear that this is the state _after_ the event has occurred.

Reply:
Fixed!